python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -1209,6 +1210,237 @@ def bitwise_not(x, out=None, **kwargs):
     return _unary_func_helper(x, _npi.bitwise_not, _np.bitwise_not, out=out, **kwargs)
 
 
+@set_module('mxnet.symbol.numpy')
+def copyto(dst, src, casting='same_kind', where=True):  # pylint: disable=W0621
+    r"""
+    Copies values from one array to another, broadcasting as necessary.
+
+    Raises a TypeError if the `casting` rule is violated, and if
+    `where` is provided, it selects which elements to copy.
+
+    Parameters
+    ----------
+    dst : ndarray
+        The array into which values are copied.
+    src : ndarray
+        The array from which values are copied.
+    casting : {'no', 'equiv', 'safe', 'same_kind', 'unsafe'}, optional
+        Controls what kind of data casting may occur when copying.
+
+        * 'no' means the data types should not be cast at all.
+        * 'equiv' means only byte-order changes are allowed.
+        * 'safe' means only casts which can preserve values are allowed.
+        * 'same_kind' means only safe casts or casts within a kind,
+            like float64 to float32, are allowed.
+        * 'unsafe' means any data conversions may be done.
+    where : array_like of bool, optional
+        A boolean array which is broadcasted to match the dimensions
+        of `dst`, and selects elements to copy from `src` to `dst`
+        wherever it contains the value True.
+    """
+
+    if not isinstance(src, numeric_types):
+        src.astype(dst.dtype, casting=casting)
+        if not isinstance(where, numeric_types):
+            _npi.copy2(dst, src, where, src=None,
+                       casting=casting, where=None)
+        else:
+            _npi.copy2(dst, src, src=None, casting=casting, where=where)
+    else:
+        if not isinstance(where, numeric_types):
+            _npi.copy2(dst, where, src=src, casting=casting, where=None)
+        else:
+            _npi.copy2(dst, src=src, casting=casting, where=where)
+
+
+@set_module('mxnet.symbol.numpy')
+def nanargmin(a, axis=None):
+    """
+    Return the indices of the minimum values in the specified axis ignoring
+    NaNs. For all-NaN slices ``ValueError`` is raised. Warning: the results
+    cannot be trusted if a slice contains only NaNs and Infs.
+    Parameters
+    ----------
+    a : ndarray
+        Input data.
+    axis : int, optional
+        Axis along which to operate.  By default flattened input is used.
+    Returns
+    -------
+    index_array : ndarray
+        An array of indices or a single index value.
+    See Also
+    --------
+    argmin, nanargmax
+    Examples
+    --------
+    >>> a = np.array([[np.nan, 4], [2, 3]])
+    >>> np.argmin(a)
+    0
+    >>> np.nanargmin(a)
+    2
+    >>> np.nanargmin(a, axis=0)
+    array([1, 1])
+    >>> np.nanargmin(a, axis=1)
+    array([1, 0])
+    """
+    mask = isnan(a)
+    if mask is not None:
+        copyto(a, 2147483647, where=mask)
```

Comment:
2147483647 seems to be correct for int32 dtype. But you're testing for 'float16', 'float32', 'float64'. How is this supposed to work?